### ISSUE TITLE
Evidences

### ISSUE THREAD

**Issue by DeniseSl22** [PathVisio/pathvisio4-ant#13]
I can click on the "query pubmed" button, without actually adding an ID (I believe this should be added in the "value" box, right?)
![image](https://user-images.githubusercontent.com/26277832/169982564-005317ce-1b94-479c-9f7a-618753c00c7b.png)
Also, this is not consistent with the "citation" property, where I cab query a publication based on something I add in the "identifier" box.

**Comment by DeniseSl22** [comment]
Again, URL iso Url

**Comment by DeniseSl22** [comment]
The term "value" here is confusing to me, what does it mean? And what do the * stand for? Mandatory values?

**Comment by DeniseSl22** [comment]
I would rephrase this error message: 
![image](https://user-images.githubusercontent.com/26277832/169983260-ccaca68f-1308-4dba-8210-9f97864eb50e.png)

**Comment by DeniseSl22** [comment]
> I would rephrase this error message: ![image](https://user-images.githubusercontent.com/26277832/169983260-ccaca68f-1308-4dba-8210-9f97864eb50e.png)

Actually, with this combination of input (so adding a value, no ID, and selecting a database), I get two error messages, the one above, and when I close that one, I get another one: 
![image](https://user-images.githubusercontent.com/26277832/169983564-c52b4a03-0df3-429c-b525-c1f31283765a.png)

**Comment by mkutmon** [comment]
as in forms generally (*) indicates mandatory items.

**Comment by mkutmon** [comment]
I agree with the comment about the name "Value" - maybe "Term" would be better?

**Comment by Finterly** [comment]
Commits 9931f5e  and  ffee76b resolves most of these issues.
